lisa/util/shell.py

```diff
@@ -261,12 +261,15 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         stdout_content = stdout.readline()
         stdout.close()
 
+        self.is_bash = True
         if stdout_content and "Windows" in stdout_content:
             self.is_posix = False
             shell_type = WindowsShellType()
         else:
             self.is_posix = True
             shell_type = spur.ssh.ShellTypes.sh
+            if stdout_content and "Unknown syntax" in stdout_content:
```

Comment:
How about assign shell_type to minimal here, and use the variable called `_shell_type`?

Reply:
_shell_type is used for?
I have added some changes, but have an error. I'm not sure how to fix that, can you help?

Reply:
If the selected shell type doesn't need to be used in next lines, it doesn't need have a variable.